src/test/helpers.js

```diff
@@ -1,29 +1,55 @@
+const fl = require('../core/flNames')
+
 const id =
   x => x
 
 const slice =
   x => Array.prototype.slice.call(x)
 
+const contains = xs => x =>
+  xs.indexOf(x) !== -1
```

Comment:
given that this is a helper, do we care that xs could be null here? I am thinking no because being that its a test, it's going to very publicly break

Reply:
I agree with you.